go.mod

```diff
@@ -47,3 +48,5 @@ require (
 )
 
 replace github.com/gogo/protobuf => github.com/ElrondNetwork/protobuf v1.3.2
+
+replace github.com/ElrondNetwork/arwen-wasm-vm/1_2_7 => github.com/ElrondNetwork/arwen-wasm-vm v1.2.7
```

Comment:
please remove this

Reply:
Replaced with `arwen-was-vm/v1_3`, which works correctly now.